dts/arm/nxp/nxp_rt1170_cm7.dtsi

```diff
@@ -48,7 +48,8 @@
 		gpio2: gpio@42008000 {
 			compatible = "nxp,imx-gpio";
 			reg = <0x42008000 0x4000>;
-			interrupts = <102 0>, <103 0>;
+			interrupts = <99 0>;
+			label = "GPIO_2";
```

Comment:
remove `label` prop

Reply:
This change for IRQs should be in a separate commit.

Reply:
@galak, when do you think it is time to add a CI check for the label prop?

Reply:
Will change in another commi